libs/api/domains/education/src/lib/education.service.ts

```diff
@@ -24,7 +45,105 @@ export class EducationService {
   async downloadPdfLicense(
     nationalId: string,
     licenseId: string,
-  ): Promise<Response> {
-    return this.mmsApi.downloadLicensePDF(nationalId, licenseId)
+  ): Promise<string | null> {
+    const responseStream = await this.mmsApi.downloadLicensePDF(
+      nationalId,
+      licenseId,
+    )
+
+    return this.s3Service.uploadFileFromStream(responseStream, {
+      fileName: uuid(),
+      bucket: this.config.fileDownloadBucket,
+    })
+  }
+
+  async getFamily(nationalId: string) {
+    const family = await this.nationalRegistryApi.getMyFamily(nationalId)
+    return family.filter(
+      (familyMember) =>
+        nationalId === familyMember.Kennitala ||
+        (!['1', '2'].includes(familyMember.Kyn) &&
```

Comment:
There are more variations of `Kyn`. Probably missing `7`???

Reply:
Ah, yes 7 is missing, I will add it here.
It is not a part of the nationaregistry api domain either, we will need to fix that in the next slice as well.